themes/gfsc/layouts/our-work/list.html

```diff
@@ -23,7 +23,7 @@
           {{ end }}
         {{ end }}
         {{ if (gt $project_count 0 ) }}
-          <li class="work__filter work__filter--hidden">
+          <li class="work__filter" style="display: none">
```

Comment:
I don't understand why this is necessary or relevant to this PR - i think it's much cleaner to leave as is!

Reply:
I think (and I could be wrong here) if you just move the `&__filter--hidden` section within the sass file below the `&__filter` section this should fix this without having to change the js

Reply:
Not sure how much you know about css hierarchies but there are a lot of interesting rules about it's applied. The first and simplest rule tho is that styles cascade (cascading style sheets, innit!) so styles at the bottom of the sheet get applied after styles at the top https://developer.mozilla.org/en-US/docs/Learn/CSS/Building_blocks/Cascade_and_inheritance#specificity_2

Reply:
i will give it a go! (less code is always better than more, right?)

seems to have done the trick. putting back in old code